models/bootstrap.go

```diff
@@ -1,7 +1,7 @@
 package models
 
-func bootstrapSQL() [104]string {
-	sql := [104]string{
+func bootstrapSQL() [105]string {
+	sql := [105]string{
```

Comment:
This should be 106 if you're adding two rows in?

Reply:
This does indeed require 106 to avoid models/bootstrap.go:110:3: array index 105 out of bounds [0:105]

Reply:
Successfully tested locally following changes.. schema created and populated as expected